pkg/api/v20191231preview/openshiftcluster_validatestatic.go

```diff
@@ -184,6 +184,10 @@ func (sv openShiftClusterStaticValidator) validateServicePrincipalProfile(path s
 }
 
 func (sv openShiftClusterStaticValidator) validateNetworkProfile(path string, np *NetworkProfile) error {
+	if np.PodCIDR == api.MachineCIDR || np.ServiceCIDR == api.MachineCIDR {
```

Comment:
Once the above const Machine CIDR is fixed we may need to change this condition to appropriate CIDR name.

Reply:
Done